sdk/src/client/node_api/plugin/mod.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Custom plugin call
+
+use core::str::FromStr;
+
+use reqwest::Method;
+
+use crate::client::{ClientInner, Result};
+
+impl ClientInner {
+    /// Extension method which provides request methods for plugins.
+    pub async fn call_plugin_route<T>(
+        &self,
+        base_plugin_path: &str,
+        method: &str,
+        method_path: &str,
+        query_params: Vec<String>,
+        request_object: Option<String>,
+    ) -> Result<T>
+    where
+        T: serde::de::DeserializeOwned + std::fmt::Debug + serde::Serialize,
+    {
+        let mut method = method.to_string();
+        method.make_ascii_uppercase();
+
+        let req_method = reqwest::Method::from_str(&method);
+
+        let node_manager = self.node_manager.read().await;
+        let path = format!("{}{}{}", base_plugin_path, method_path, query_params.join("&"));
+        let timeout = self.get_timeout().await;
+
+        match req_method {
+            Ok(Method::GET) => node_manager.get_request(&path, None, timeout, false, false).await,
+            Ok(Method::POST) => {
+                let pow = self.get_local_pow().await;
+                node_manager
+                    .post_request_json(&path, timeout, request_object.into(), pow)
```

Comment:
The pow bool here is to specify if for this request nodes should be used that support remote PoW or not
I think this should be just hardcoded to true
```suggestion
                node_manager
                    .post_request_json(&path, timeout, request_object.into(), true)
```